mathics/builtin/algebra.py

```diff
@@ -178,6 +178,26 @@ def find_vars(e, e_sympy):
     return variables
 
 
+def exponent(expr, var):
+    f = expr.to_sympy()
+    x = var.to_sympy()
+    list = [i.as_coeff_exponent(x) for i in f.expand(power_exp=False).as_ordered_terms()]
```

Comment:
This could be cleaned up a bit, no need to allocate a new list here. Also, `list` isn't a good variable name.